app/src/main/java/com/lastaoutdoor/lasta/models/user/UserModel.kt

```diff
@@ -39,4 +42,41 @@ data class UserModel(
         friendRequests,
         favorites)
   }
+
+  // Returns the description text for the preferred activity and level
+  @Composable
+  fun descrText(): String {
```

Comment:
This function could otimized like this: 
```suggestion
  @Composable
  fun descrText(): String {
    val prefActivity = this.prefActivity
    val prefActivityLevel =
        when (prefActivity) {
          ActivityType.CLIMBING -> this.levels.climbingLevel
          ActivityType.HIKING -> this.levels.hikingLevel
          ActivityType.BIKING -> this.levels.bikingLevel
        }

      return prefActivity.resourcesToString(LocalContext.current) + " " + prefActivityLevel.resourcesToString(LocalContext.current)
```

string concatenation could replace the duplicate code. This will need Strings adaptation in strings.xml file too!

Reply:
Yeah you're right, @CassioManuguerra wrote it and I juste used it without re-thinking about it